<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGBD_Camera_Pair_Registration</class>
 <widget class="qMRMLWidget" name="RGBD_Camera_Pair_Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>545</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sequence Browser:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="sequenceBrowser">
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLSequenceBrowserNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="computeCameraTransformsCollapsibleButton">
     <property name="text">
      <string>Compute Camera Transforms</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ctkDirectoryButton" name="DirectoryButton"/>
      </item>
      <item>
       <widget class="QComboBox" name="videoIDComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="rightOrAboveComboBox">
        <item>
         <property name="text">
          <string>RIGHT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ABOVE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="depthToRASPushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Run the algorithm.</string>
        </property>
        <property name="text">
         <string>Get Depth to RAS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="registerToolBoundingBoxCollapisbleButton">
     <property name="text">
      <string>Register Tool Bounding Box</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>RIGHT Camera Transform:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="rightCameraTransform">
          <property name="nodeTypes">
           <stringlist notr="true">
            <string>vtkMRMLLinearTransformNode</string>
            <string>vtkMRMLTransformNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ABOVE Camera Transform:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="aboveCameraTransform">
          <property name="nodeTypes">
           <stringlist notr="true">
            <string>vtkMRMLLinearTransformNode</string>
            <string>vtkMRMLTransformNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>RIGHT Bounding Box Sequence:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="rightBoundingBoxSequence">
          <property name="nodeTypes">
           <stringlist notr="true">
            <string>vtkMRMLSequenceNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ABOVE Bounding Box Sequence:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="aboveBoundingBoxSequence">
          <property name="nodeTypes">
           <stringlist notr="true">
            <string>vtkMRMLSequenceNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="registerBoundingBoxButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
